### ISSUE TITLE
Crash with "No static method lineHeight" exception

### ISSUE THREAD

**Issue by Occhioverde** [MM2-0/Kvaesitso#156]
Good morning.

As per the title, after the update to the latest (1.14.5) version, Kvæsitso started crashing every time I try to open it reporting the following error:

```
No static method lineHeight(Landroid/graphics/Paint$FontMetricsInt;)l in class Lorg/apache/http/util/Encodi...
```

Sorry if the message is missing a part, but the notification only displays a small portion of the text and trying to open the report only leads to another crash with the same exception.

For the same reason I can't provide you with the device info that usually the app prints after a crash, but in case it can be useful to debug the problem the devices on which I'm encountering the issue are a Huawei BAH2-W19 running Android 8.0.0 and an Asus Zenfone 4 Max with the same OS version.

Thanks in advance for the help.

**Comment by MM2-0** [comment]
Thanks for the report. Could you please open the path `Internal Storage/Android/data/de.mm20.launcher2.release/files/crashReports` in a file manager and post the contents of the most recent file here?

**Comment by MM2-0** [comment]
This is so weird. For some reason, when I enable optimizations, the compiler somehow messes things up so badly that it crashes. But only on older Android versions. I don't know why this is happening, need more time to investigate.

Anyway, this APK with code optimizations disabled should work for now:
[app-release.zip](https://github.com/MM2-0/Kvaesitso/files/9427987/app-release.zip)

**Comment by Occhioverde** [comment]
> this APK with code optimizations disabled should work for now

Yes, I can confirm that this build is working just fine on both the devices I mentioned before. Thank you very much!

I remain available in case you need further debugging info.

**Comment by MM2-0** [comment]
I think the issue should be fixed in 019afb3 so there shouldn't be any issues with 1.14.6 anymore.
